fix(content) Remnant-Drak: Fixing the Archon and Dodging challenge (#7239)
endless-sky/endless-sky: data/map.txt

	tribute 1300
		threshold 3000
		fleet "Large Core Pirates" 16
		
planet Crataegusa
	attributes "gas giant" "requires: gaslining" uninhabited
	landscape land/nasa30

> 	description "At a distance, Crataegusa appears utterly linear with its shear cloud lines circling the planet. Beneath the surface, however, the atmosphere is a turbulent maelstrom of cyclonic winds at odds with the surface appearance."

Reviewer: ```suggestion
	description "At a distance, Crataegusa has a plain appearance, with linear shear cloud lines circling the planet. Beneath the surface, however, the atmosphere is a turbulent maelstrom of cyclonic winds at odds with the surface appearance."
```

> 	description "	As you skim through the clouds your sensors report intense energy fluctuations from all directions, some of which echoes the readings from the wormholes."

Reviewer 2: ```suggestion
	description "	As you skim through the clouds your sensors report intense energy fluctuations from all directions, some of which echo the readings from the wormholes."
```
